app/lib/actions/katello/repository/create.rb

```diff
@@ -33,6 +34,12 @@ def plan(repository, args = {})
 
             concurrence do
               plan_self(:repository_id => repository.id, :clone => clone)
+              repository.product.alternate_content_sources.with_type(repository.content_type).each do |acs|
+                acs.smart_proxies.each do |smart_proxy|
+                  smart_proxy_acs = ::Katello::SmartProxyAlternateContentSource.create(alternate_content_source_id: acs.id, smart_proxy_id: smart_proxy.id, repository_id: repository.id)
+                  plan_action(Pulp3::Orchestration::AlternateContentSource::Create, smart_proxy_acs)
```

Comment:
We should verify existence of repo.url here as well before creating ACSs for new repos.